docs/intro/quickstart.md

```diff
@@ -0,0 +1,98 @@
+> - Running your app
+
+<br/>
+The steps to build the app are as follows:
+
+1. [Clone or create project](#clone-project)
+2. [Building UI](#build-ui)
+3. [Customize style](#customize-style)
+4. [Add state](#add-state)
+5. [Run the app](#run-app)
+
+## 1. Clone or create project {#clone-project}
+
+To kick off your project, the first step is to [create a new project](#). However, to make things easier, we've already created a starter app for you. Simply open [this link](https://app.flutterflow.io/project/f-f-quick-start-app-umu392), click the '**Clone**' button, and the project will be instantly added to your account. 
+
+<img src="../../static/img/clone-project.png" alt="clone-project.png"  />
+
+After cloning the project, you’ll see a page with product images and descriptions. You’ll add a feature that allows users to update the product quantity.
+
+<img src="../../static/img/zero-to-final.png" alt="zero-to-final.png"  />
+
+## 2. Building UI {#build-ui}
+
+In this section, we'll see how to build the user interface (UI) for this feature. This involves creating a layout and adding various widgets to our page.
+
+To build the UI:
+
+1. We'll begin by adding a [Container](#) widget. This will serve as the parent widget for our buttons and labels.
+2. Within the Container, add the [Row](#) > [Text](#) widget to display the ‘Quantity’ label. The Row widget is used to arrange its children (in this case, the Text widget) in a horizontal row.
+3. Next, add another Row widget containing two [IconButtons](#). Users will interact with these to increase or decrease the product quantity. Also, include a Text widget to display the updated quantity.
+    1. Adjust the padding of the widgets to create a spacious and balanced layout.
+    2. Change the icon of the decrease button to a minus symbol.
+
+> You can learn more about creating layouts [here](#).
+
+
+<iframe src="https://demo.arcade.software/5CNFKTzhvnHPrLyZNzgZ?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="500"></iframe>
+
+## 3. Customize Style {#customize-style}
+
+The next step is to customize the style of UI elements. This includes changing the colors, fonts, and sizes of your buttons and labels. In FlutterFlow, you can do this via the [properties panel](#) which provides a range of options for customization.
+
+To customize the style:
+
+1. Adjust the height and color of the Container, and add a border radius.
+2. Change the font weight, size of the quantity label and its value.
+3. Change the color of the increase and decrease buttons to match your design.
+
+
+<iframe src="https://demo.arcade.software/MGpg8TSzMGBusCGyOk89?embed&show_copy_link=true" title="Update product quantity" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="600"></iframe>
```

Comment:
Without the steps, the Arcade is not very self explanatory. We should definitely remove steps from text and add it to Arcade callouts. Remove video shots wherever its not really adding weight to the step. For example, typing movement makes more sense for a video shot, but otherwise screenshots in Arcade is okay just for showing properties & its value.